src/bento/dashboardData.js

```diff
@@ -1,6 +1,13 @@
 /* eslint-disable */
 import gql from 'graphql-tag';
 
+// --------------- Dashboard Data --------------------
+export const dashboardData = {
```

Comment:
can you rename "dashboardData" to themeToggleTooltip ?

Reply:
done